_overviews/toolkit/sttp-make-request.md

```diff
@@ -0,0 +1,40 @@
+---
+title: How to make an HTTP request?
+type: section
+description: How to make any HTTP request with Scala Toolkit.
+num: 13
+previous-page: sttp-intro
+next-page: sttp-variable-urls
+---
+
+{% include markdown.html path="_markdown/install-sttp.md" %}
+
+## Make a request
+The simplest way to make a request with Scala Toolkit's http client library, sttp, is to use the `SimpleHttpClient` instance. 
+You need to import a few members from the library, and then write the code that makes the request:
+
+```scala
+import sttp.client3.{SimpleHttpClient, UriContext, basicRequest}
+
+val client = SimpleHttpClient() // Create the instance of SimpleHttpClient
+val request = basicRequest.get(uri"https://httpbin.org/get") // Construct a get request to an example service - https://httpbin.org/get
+val response = client.send(request) // send the request and get the response
+println(response.body) // print the body of the response
+```
+
+## Other request types
+You can swap `basicRequest.get` with any HTTP method you want. For example:
+```scala
+val request = basicRequest.post(uri"https://httpbin.org/get")
```

Comment:
maybe it wouldn't be too much of a stretch to add a string body here? so the example could be:

```
val request = basicRequest.post(uri"https://httpbin.org/post").body("Hello, world!")
```